### PR TITLE
src/components: add BannerAppButtons component with component test

### PR DESCRIPTION
**Pull request by maceksimon**
* Add `BannerAppButtons.vue` rendering links to RTWBB app download options
* Add `BannerAppButtons.cy.js` tests
* Add `colorWhiteOpacity` and `borderRadiusCardSmall` to global config for styling the banner background
* Add `gap-16` to global CSS utility classes for styling the button layout
* Add translations for `login.bannerAppButtons`
* Incorporate `BannerAppButtons` into `FormLogin` component

### CODE REVIEW

File: src/components/__tests__/BannerAppButtons.cy.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      cy.mount(BannerAppButtons, {
+        props: {},
+      });
+      cy.viewport('macbook-16');
+    });
+
+    it('renders container with semi-transparent white background and rounded corners', () => {
+      cy.dataCy('banner-app-buttons')
+        .should('have.css', 'padding', '16px')
+        .should('have.backgroundColor', 'rgba(255, 255, 255, 0.5)')
```
Comment (tmszi) [inline src/components/__tests__/BannerAppButtons.cy.js:28]: Same as in previous PR https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/102#discussion_r1377247255

File: src/components/__tests__/BannerAppButtons.cy.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        .should('have.css', 'border-radius', '8px');
+    });
+
+    it('renders title', () => {
+      cy.dataCy('banner-app-buttons-title')
+        .should('have.css', 'font-size', '14px')
+        .should('have.css', 'font-weight', '700')
```
Comment (tmszi) [inline src/components/__tests__/BannerAppButtons.cy.js:35]: Same as in previous PR https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/102#discussion_r1377247255.

File: src/components/__tests__/BannerAppButtons.cy.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+          expect($title.text()).to.equal(
+            i18n.global.t('login.bannerAppButtons.title'),
+          );
+        });
+    });
+
+    it('renders description', () => {
+      cy.dataCy('banner-app-buttons-description')
+        .should('have.css', 'font-size', '12px')
+        .should('have.css', 'font-weight', '400')
```
Comment (tmszi) [inline src/components/__tests__/BannerAppButtons.cy.js:48]: Same as in previous PR https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/102#discussion_r1377247255.

File: src/components/__tests__/BannerAppButtons.cy.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+          expect($description.text()).to.equal(
+            i18n.global.t('login.bannerAppButtons.description'),
+          );
+        });
+    });
+
+    it('uses correct spacing', () => {
+      cy.dataCy('banner-app-buttons-title')
+        .should('have.css', 'margin-top', '0px')
+        .should('have.css', 'margin-bottom', '0px');
```
Comment (tmszi) [inline src/components/__tests__/BannerAppButtons.cy.js:61]: Same as in previous PR https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/102#discussion_r1377247255.

File: src/components/__tests__/BannerAppButtons.cy.js [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+      cy.dataCy('banner-app-buttons-description')
+        .should('have.css', 'margin-top', '4px')
+        .should('have.css', 'margin-bottom', '16px');
```
Comment (tmszi) [inline src/components/__tests__/BannerAppButtons.cy.js:65]: Same as in previous PR https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/102#discussion_r1377247255.

File: src/components/__tests__/BannerAppButtons.cy.js [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+        .should('be.visible')
+        .then(($img) => {
+          cy.testImageHeight($img);
+        });
+    });
+
+    it('renders button in a flexbox with gap', () => {
+      cy.dataCy('banner-app-buttons-container')
+        .should('have.css', 'display', 'flex')
+        .should('have.css', 'flex-wrap', 'wrap')
```
Comment (tmszi) [inline src/components/__tests__/BannerAppButtons.cy.js:97]: Same as in previous PR https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/102#discussion_r1377247255.

File: src/components/__tests__/BannerAppButtons.cy.js (hunk 1 above)
Comment (maceksimon) [reply src/components/__tests__/BannerAppButtons.cy.js]: Edited.

File: src/components/__tests__/BannerAppButtons.cy.js (hunk 2 above)
Comment (maceksimon) [reply src/components/__tests__/BannerAppButtons.cy.js]: Edited.

File: src/components/__tests__/BannerAppButtons.cy.js (hunk 3 above)
Comment (maceksimon) [reply src/components/__tests__/BannerAppButtons.cy.js]: Edited.

File: src/components/__tests__/BannerAppButtons.cy.js (hunk 4 above)
Comment (maceksimon) [reply src/components/__tests__/BannerAppButtons.cy.js]: Edited.

File: src/components/__tests__/BannerAppButtons.cy.js (hunk 5 above)
Comment (maceksimon) [reply src/components/__tests__/BannerAppButtons.cy.js]: Edited.

File: src/components/__tests__/BannerAppButtons.cy.js (hunk 6 above)
Comment (maceksimon) [reply src/components/__tests__/BannerAppButtons.cy.js]: Edited.

**Review by tmszi — APPROVED** [review]

**Pull request merged**